R/get_code.R

```diff
@@ -102,6 +144,7 @@ get_code.default <- function(x,
                              deparse = FALSE,
                              files_path = NULL,
                              dataname = NULL,
+                             names = NULL,
```

Comment:
Wait, this is necessary? :thinking:

Reply:
I think everything existing in other methods needs to exist in the default method

Reply:
maybe it actually doesnt

Reply:
b06b294f0988d5d3eb92582f1e5c1545b2f048a3

Reply:
> I think everything existing in other methods needs to exist in the default method

I don't think so :thinking:  :monocle_face: